public/components/CampaignAnalytics/Analytics/CampaignUniquesChart.js

```diff
@@ -9,16 +9,17 @@ export default class CampaignUniquesChart extends React.Component {
 
   render () {
 
-    console.log(this.props.data);
-
-    let data = this.props.data.map( (item) => {
+    let data = [];
+    if (this.props.data) {
```

Comment:
I'm not sure we want to show the graph if there is no data. Can we add:
```
if (!this.props.data) return null;
```
as the first line of the render method instead?